src/pyclass.rs

```diff
@@ -378,7 +380,6 @@ impl PyTypeBuilder {
             itemsize: 0,
             // `c_ulong` and `c_uint` have the same size
             // on some platforms (like windows)
-            #[allow(clippy::useless_conversion)]
```

Comment:
this isn't working for some reason

Reply:
Nope - still getting this on stable clippy.

Reply:
Hmm if moving it up resolves, I guess we'll live with it.